docs/application/web/tutorials/app-filtering.md

```diff
@@ -53,29 +53,29 @@ thus prevents incompatible applications from being installed.
 ![Feature-based filtering](./media/app_filtering_basic_flow.png)
```

Comment:
I have a question about this app_filtering_basic_flow.png.
The similar image is used in other profiles also,
and it includes "Tizen Store" in it.

Can you modify the images?
If you can, it would be better to change the "Tizen Store", too.
If you can't, for now, it would be kept.

Reply:
![app_filtering_basic_flow_updated](https://user-images.githubusercontent.com/99946113/203689253-1810785a-3705-40d4-bf33-23c116ec5694.png)
@mijin85cho would this be okay as a replacement image?

Reply:
@safir-srbd Thanks for the suggestion.
We had some talk about your suggestion.
"Official App site" sounds a little bit vague,
nevertheless "Application Store" reminds of another application store of other OS.
So we are considering "Tizen application Store".
Is it possible to try that it?

Reply:
@mijin85cho Sure I will replace it with "Tizen Application Store" and then commit it to this PR.

Reply:
I appreciate your work!
However, was "Tizen application store" too long for the image?
As the "App Store" can bring up the image of other official "App Store",
if possible, I hope to avoid it.

Reply:
@mijin85cho No worries, I can use application instead of app. At first, it just felt too long, but I can see how using "app store" could become an issue. I will make the changes and push another commit.